tests/test_roledb.py

```diff
@@ -107,20 +109,23 @@ def test_clear_roledb(self):
     # Test for an empty roledb, a length of 1 after adding a key, and finally
     # an empty roledb after calling 'clear_roledb()'.
     self.assertEqual(0, len(tuf.roledb._roledb_dict['default']))
-    tuf.roledb._roledb_dict['default']['Root'] = {'keyids': ['123'], 'threshold': 1}
+    tuf.roledb._roledb_dict['default']['Root'] =\
+        securesystemslib.util.load_json_file(os.path.join(self.test_data_path,
+        "root.json"))['signed']
```

Comment:
Style tips, for pedagogical purposes (NBD):
- space between `=` and `\`, per our coding guidelines, for readability
- Ideally, avoid `\` continuation.  It's reasonable here, since there aren't enough characters left to get to the `(`, and since adding a set of `()` might make it a tiny bit harder to read.